banners/english/C24_WMDE_Desktop_EN_02/styles/styles.scss

```diff
@@ -0,0 +1,57 @@
+// This is the file where we import the theme-specific component styles
+@use 'src/themes/Treedip/swatches/skin_vector-2022' with (
```

Comment:
When https://github.com/wmde/fundraising-banners/pull/579 is merged the payment icons here will break. You should be able to add `$select-group-image-label: true,` to the list already to stop that.

Reply:
added that line

Reply:
Oh, I expected SASS would ignore an undefined parameter as the IDE doesn't throw warnings, but apparently the compiler doesn't like it.

Reply:
removed the line and will add it back later, when #579 is actually merged